test/unit/slinksService.spec.js

```diff
@@ -0,0 +1,48 @@
+describe('SlinksService', function() {
+  beforeEach(module('slinksApp'));
+
+  var SlinksService, httpBackend;
+
+  var slinksData = {
+    messages: {
+      matches: [
+        {
+          text: "<https://slack.com/>",
+          previous: {
```

Comment:
slinksService.spec.js contains duplicate code to slinksController.spec.js. Could it be extracted to a helper?